cellmlmanip/model.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, name: str, model: 'Model') -> None:
         self.siblings: Set[str] = set()
         self.encapsulated: Set[str] = set()
 
-        self.variables: Dict[str, Dict] = {}
+        self.variables: OrderedDict[str, Dict] = OrderedDict()
```

Comment:
I think you might want `Dict` instead of `OrderedDict` in the type, judging by https://docs.python.org/3/library/typing.html#typing.Dict. (In Python 3.7 you can use `typing.OrderedDict`...)

Reply:
Cheers!